index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img class="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <nav>
+        <a href=""
+          ><img
+            class="nav-logo"
+            src="./images/header/Hamburger_icon.svg.png"
+            alt="logo"
+        /></a>
+      </nav>
+    </header>
+    <main class="headline">
+      <section>
+        <div class="name"><h1>I’m Ali Hassan. Glad to see you!</h1></div>
+        <div class="introduction">
+          <p>
+            I'm a software developer! I can help you build a product, feature or
+            website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <div class="connect">
+          <p>LET'S CONNECT</p>
+        </div>
+        <div>
+          <ul class="social">
+            <li>
+              <a href="#"
+                ><img src="./images/main/Vector-3.png" alt="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/main/Linkedin icon.svg" alt="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/M.svg" alt="M" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/Lomri.svg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/main/V.svg" alt="V" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+  </body>
```

Comment:
- [ ] I noticed a horizontal scroll effect in your current design, which wasn't present initially. This could impact the overall user experience and needs to be addressed. So kindly fix this as well :+1: I have attached a video reference below. :wink: 

- Video Reference

[Screencast from 06-03-2023 07:44:45 AM.webm](https://github.com/mhassa204/Portfolio-Project/assets/102699958/f21d2c2b-950c-4998-bf49-a2b97af8103a)